spec/welcome/index_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "welcome index page", type: :feature do
+  it "can see welcome message" do
+
+    visit "/"
+
+    expect(page).to have_content("Welcome To Gard(in) That!")
```

Comment:
we can add the tests for the nav bar once we have more in our `application.html`